package.json

```diff
@@ -49,5 +49,6 @@
     "nodemon": "^2.0.19",
     "superagent": "^8.1.2",
     "supertest": "^6.3.3"
-  }
+  },
+  "packageManager": "yarn@1.22.22+sha512.a6b2f7906b721bba3d67d4aff083df04dad64c399707841b7acf00f6b133b7ac24255f2652fa22ae3534329dc6180534e98d17432037ff6fd140556e2bb3137e"
```

Comment:
Was this automatically added?

Reply:
Yes, I can back it out if it doesn't seem correct!

Reply:
Interesting. I haven't seen this before here so I was curious. I don't know if this is correct or not. I had thought [this line would dictate the yarn version](https://github.com/DataBiosphere/bard/blob/3dda8dfb1119a46ea9f818fae642ea33f643576e/.yarnrc#L5).